.env.example

```diff
@@ -15,4 +15,5 @@ ACHIVE_NODE_zksyncsepolia=https://zksync-sepolia.g.alchemy.com/v2/<YOUR_KEY_HERE
 ARCHIVE_NODE_zksyncmainnet=https://open-platform.nodereal.io/<YOUR_KEY_HERE>/zksync
 ARCHIVE_NODE_opsepolia=https://sepolia.optimism.io
 ARCHIVE_NODE_opmainnet=https://opt-mainnet.nodereal.io/v1/<YOUR_KEY_HERE>
-ARCHIVE_NODE_basesepolia=https://base-sepolia.blastapi.io/<YOUR_KEY_HERE>
\ No newline at end of file
+ARCHIVE_NODE_basesepolia=https://sepolia.base.org
+ARCHIVE_NODE_basemainnet=https://mainnet.base.org
```

Comment:
```suggestion
ARCHIVE_NODE_basemainnet=https://open-platform.nodereal.io/<YOUR_KEY_HERE>/base
```

Reply:
Nodereal RPC node is a full node, so it only supports the recent 129 block query. For simulations, I suppose mainnet.base.org is enough. Otherwise, we should suggest an alternative